daml/ContingentClaims/Math/Stochastic.daml

```diff
@@ -15,7 +15,8 @@ module ContingentClaims.Math.Stochastic (
   , unitIdentity
 ) where
 
-import ContingentClaims.Claim
+import ContingentClaims.Inequality(Inequality(..))
+import ContingentClaims.Internal.Claim(Claim(..))
```

Comment:
How do you feel about having these two modules instead of a single `Claim` module? I thought about it for a while and in the end I put `Inequality` in `Claim` exactly for this reason: that you'll always have an extra import.

Reply:
I have no strong opinion on this. 

I feel that `Inequality`s belong in a separate module as they are semantically different to `Claim`s. On the other hand, it does add the extra import which is not nice.

Reply:
Let's keep them in the same module for ease of imports.